test/linalg/conjarray.jl

```diff
@@ -0,0 +1,23 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+@testset "RowVector" begin
```

Comment:
Right, thanks for the heads-up. The behavior of Base tests is quite different to what happens with packages, but I had seen an issue (from memory) about using `@testset` more so I used my usual style and hoped things would be picked up.

(is the hope to unify the package and base approaches in the future?)